src/translate/translateController.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+
+export default angular.module('translate')
+    .controller('TranslateController', ['$translate', function ($translate) {
+        // https://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript?page=1&tab=votes#tab-top
```

Comment:
Please refactor this

Reply:
Updated.

Reply:
What is the point of this code? Are we keeping language selection in URL? I think it's better to store user preference in Local Storage